### ISSUE TITLE
Go 1.7.1 - go get github.com/labstack/armor/cmd/armor fails

### ISSUE THREAD

**Issue by scottjbarr** [labstack/armor#2]
When installing via `go get` the build fails.

```
$ go get github.com/labstack/armor/cmd/armor
# github.com/labstack/armor/plugin
../go/src/github.com/labstack/armor/plugin/nats.go:58: cannot use body (type func() io.Reader) as type io.Reader in argument to ioutil.ReadAll:
	func() io.Reader does not implement io.Reader (missing Read method)
../go/src/github.com/labstack/armor/plugin/proxy.go:93: req.Header.Get undefined (type func() engine.Header has no field or method Get)
../go/src/github.com/labstack/armor/plugin/proxy.go:94: cannot use res (type engine.Response) as type http.ResponseWriter in argument to p.wsProxy(t).ServeHTTP:
	engine.Response does not implement http.ResponseWriter (wrong type for Header method)
		have Header() engine.Header
		want Header() http.Header
../go/src/github.com/labstack/armor/plugin/proxy.go:94: http.Request.Body is a field, not a method
../go/src/github.com/labstack/armor/plugin/proxy.go:94: cannot use req (type engine.Request) as type *http.Request in argument to p.wsProxy(t).ServeHTTP
../go/src/github.com/labstack/armor/plugin/proxy.go:96: cannot use res (type engine.Response) as type http.ResponseWriter in argument to p.httpProxy(t).ServeHTTP:
	engine.Response does not implement http.ResponseWriter (wrong type for Header method)
		have Header() engine.Header
		want Header() http.Header
../go/src/github.com/labstack/armor/plugin/proxy.go:96: http.Request.Body is a field, not a method
../go/src/github.com/labstack/armor/plugin/proxy.go:96: cannot use req (type engine.Request) as type *http.Request in argument to p.httpProxy(t).ServeHTTP
../go/src/github.com/labstack/armor/plugin/redirect.go:90: undefined: middleware.HTTPSNonWWWRedirectWithConfig
```

**Comment by vishr** [comment]
For now, `go get` directly won't work as it is using Echo v3, which is not release yet. Once v3 is merged into master, we should be good. For now, you can use glide to fetch dependencies and build it.

**Comment by vishr** [comment]
We now have vendor folder in git, so `go get github.com/labstack/armor/cmd/armor` should work fine.
